### PR TITLE
[Symfony 2.5] Add AddViolationToBuildViolationRector

### PR DESCRIPTION
**Pull request by samsonasik**
Fixes https://github.com/rectorphp/rector-symfony/issues/210

### CODE REVIEW

**Comment by TomasVotruba** [comment]
What is missing here?

**Comment by samsonasik** [comment]
The functionality, it possibly needs a stub to make extended property works

File: src/Rector/MethodCall/AddViolationToBuildViolationRector.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+     * @param MethodCall $node
+     */
+    public function refactor(Node $node): ?MethodCall
+    {
+        $objectType = $this->nodeTypeResolver->getType($node->var);
+        if (! $objectType instanceof ObjectType) {
+            return null;
+        }
+
+        $authorizationChecker = new ObjectType('Symfony\Component\Validator\ExecutionContext');
```
Comment (TomasVotruba) [inline src/Rector/MethodCall/AddViolationToBuildViolationRector.php:61]: This class might need a stub

**Comment by samsonasik** [comment]
@TomasVotruba implemented 🎉

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

**Comment by TomasVotruba** [comment]
LGTM, Thank you :+1:

**Pull request merged**